emoji-copy@felipeftn/emojiSearchItem.js

```diff
@@ -1,23 +1,13 @@
-const St = imports.gi.St;
-
-const PopupMenu = imports.ui.popupMenu;
+import St from 'gi://St';
+import * as PopupMenu from 'resource:///org/gnome/shell/ui/popupMenu.js';
 
 /* Import the current extension, mainly because we need to access other files */
-const ExtensionUtils = imports.misc.extensionUtils;
-const Me = ExtensionUtils.getCurrentExtension();
-const Extension = Me.imports.extension;
-const EmojiButton = Me.imports.emojiButton.EmojiButton;
-
-/* Translations */
-const textDomain = imports.gettext.domain('emoji-copy');
-const getText = textDomain.gettext;
+import * as Extension from './extension.js';
+import { EmojiButton } from './emojiButton.js';
 
-/**
- * A wrapper around a menu item, with a search entry added to it, and a second
- * menu item with a list of buttons for recently used emojis.
- */
-var EmojiSearchItem = class EmojiSearchItem {
+import { gettext as _ } from 'resource:///org/gnome/shell/extensions/js/extensions.js';
```

Comment:
This line [needs](https://gjs.guide/extensions/upgrading/gnome-shell-45.html#extensionutils) to be changed to
```js
import {gettext as _} from 'resource:///org/gnome/shell/extensions/extension.js';
```

Note that not only the `/js` part from the path is removed, but the filename `extension.js` is now also in singular instead of plural.